core-library/src/main/java/org/silverpeas/core/admin/persistence/ComponentInstanceTable.java

```diff
@@ -98,84 +98,85 @@ protected ComponentInstanceRow fetchComponentInstance(ResultSet rs) throws SQLEx
    * Returns the instance with the given id.
    * @param id the unique identifier of the component instance
    * @return the instance with the given id.
-   * @throws AdminPersistenceException
+   * @throws SQLException
    */
-  public ComponentInstanceRow getComponentInstance(int id) throws AdminPersistenceException {
+  public ComponentInstanceRow getComponentInstance(int id) throws SQLException {
     return getUniqueRow(SELECT_INSTANCE_BY_ID, id);
   }
 
-  static final private String SELECT_INSTANCE_BY_ID =
-      "select " + INSTANCE_COLUMNS + " from ST_ComponentInstance where id = ?";
+  public static final String SELECT = "select ";
+  private static final String SELECT_INSTANCE_BY_ID =
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)